docs/getting-started/dockstore-workflows.rst

```diff
@@ -127,9 +167,9 @@ removed from Dockstore.
 
 .. tip:: Since the workflows field is an array, this file supports multiple workflows on Dockstore stemming from
    the same repository on GitHub. This is useful if you store a lot of your workflows in the same GitHub
-   repository. This is achieved setting a different value for the name field for each entry.
+   repository. This is achieved setting a different value for the name field for (Corresponds to the workflow name).
```

Comment:
1. Was like this, but should be "...achieved **by** setting a different value..."
2. "(Corresponds to the workflow name)" -- is this complete? Sentence doesn't quite read right, and why is is a capital C in "Corresponds"? Is this a RST link or something that I shouldn't be reading literally?

Reply:
I was just trying to emphasize that the name field in the dockstore.yml corresponds to the workflow name property of the workflow.

Looks like I accidentally deleted "each entry"

Reply:
> This is achieved by setting a different value for the name field for each entry (corresponds to the workflow name).